Assets/SEE/DataModel/DG/IO/LSPImporter.cs

```diff
@@ -295,6 +346,37 @@ IEnumerable<string> RelevantDocumentsForPath(string path)
             }
         }
 
+        /// <summary>
+        /// Associates the <paramref name="diagnostics"/> for the file at the given <paramref name="path"/>
+        /// with the corresponding nodes in the graph. Specifically, the diagnostics are counted for each node
+        /// and stored as attributes in the nodes.
+        /// </summary>
+        /// <param name="diagnostics">The diagnostics to associate with the nodes.</param>
+        /// <param name="path">The path of the file to which the diagnostics belong.</param>
+        private void HandleDiagnostics(IEnumerable<Diagnostic> diagnostics, string path)
+        {
+            foreach (Diagnostic diagnostic in diagnostics)
+            {
+                if (diagnostic.Severity.HasValue && IncludeDiagnostics.HasFlag(diagnostic.Severity.Value.ToDiagnosticKind()))
+                {
+                    IEnumerable<Node> diagnosticNodes = FindNodesByLocation(path, Range.FromLspRange(diagnostic.Range));
+                    foreach (Node node in diagnosticNodes)
+                    {
+                        DiagnosticSeverity severity = diagnostic.Severity.Value;
+                        if (node.TryGetInt(severity.Name(), out int count))
+                        {
+                            node.SetInt(severity.Name(), count + 1);
+                        }
+                        else
+                        {
+                            // If the severity metric is not yet set, we set it to 1.
+                            node.SetInt(severity.Name(), 1);
+                        }
+                    }
+                }
+            }
+        }
+
```

Comment:
Method `HandleCallHierarchyAsync` appears to be unused.

Reply:
The method name appears in code that was commented out. So I assume it will still be needed in the future.

Reply:
Yes, there is currently a bug in OmniSharp's LSP library that prevents me from using it, but conceptually this code should work, I just need to wait for the fix.